android/tests_backend/widgets/slider.py

```diff
@@ -0,0 +1,24 @@
+from java import jclass
+
+from android.os import Build
+
+from .base import SimpleProbe
+
+
+class SliderProbe(SimpleProbe):
+    native_class = jclass("android.widget.SeekBar")
+
+    @property
+    def position(self):
+        return (self.native.getProgress() - self._min) / (self._max - self._min)
+
+    def change(self, position):
+        self.native.setProgress(self._min + round(position * (self._max - self._min)))
+
+    @property
+    def _min(self):
+        return 0 if (Build.VERSION.SDK_INT < 26) else self.native.getMin()
```

Comment:
It won't make any difference here, because Toga doesn't currently use the minimum value property: I just added this code to the probe in case that ever changes. 

But as a general principle yes, I think it would be a good idea to run Android tests against both the minimum and maximum API levels we support, and that's what I always do before a Chaquopy release. 

> If I'm understanding correctly, we would only need to _build_ it once, but we'd need to use 2 simulators?

That's correct.